packages/ui/src/app/pages/Forum/ForumMyThreads.tsx

```diff
@@ -14,16 +14,17 @@ export const ForumMyThreads = () => {
   const [page, setPage] = useState(1)
   const { threads, pageCount, isLoading } = useMyThreads({ page, threadsPerPage: 5 })
 
+  const pagination = useMemo(
+    () => !isLoading && !!pageCount && <Pagination pageCount={pageCount} handlePageChange={setPage} page={page} />,
```

Comment:
The `!!pageCount` check seems redundant here and in `Members.tsx`, seeing that `pageCount <2` is performed in the Pagination component itself. Suggestion in below comment-

Reply:
Nice catch!